src/EA.Weee.Web.Tests.Unit/Areas/Scheme/ViewModels/TransferEvidenceNotesViewModelTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.Scheme.ViewModels
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq;
+    using FluentAssertions;
+    using Web.Areas.Scheme.ViewModels;
+    using Web.ViewModels.Shared;
+    using Xunit;
+
+    public class TransferEvidenceNotesViewModelTests
+    {
+        [Fact]
+        public void TransferEvidenceNotesViewModel_ShouldImplement_IValidatableObject()
+        {
+            typeof(TransferEvidenceNotesViewModel).Should().Implement<IValidatableObject>();
+        }
+
+        [Fact]
+        public void TransferEvidenceNotesViewModel_Constructor_ShouldInitialiseLists()
+        {
+            //act
+            var result = new TransferEvidenceNotesViewModel();
+
+            //assert
+            result.EvidenceNotesDataList.Should().NotBeNull();
+            result.SelectedEvidenceNotePairs.Should().NotBeNull();
+            result.CategoryValues.Should().NotBeNull();
+        }
+
+        [Fact]
+        public void Validate_GivenNoNotesSelected_ValidationResultExpected()
+        {
+            //arrange
+            var result = new TransferEvidenceNotesViewModel();
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>());
+
+            //act
+            var validationResults = result.Validate(new ValidationContext(result));
+
+            //assert
+            validationResults.Should().Contain(v =>
+                v.ErrorMessage.Equals("Select at least one evidence note to transfer from")
+                && v.MemberNames.Contains("SelectedEvidenceNotePairs"));
+        }
+
+        [Fact]
+        public void Validate_GivenMoreThanFiveNotesSelected_ValidationResultExpected()
+        {
+            //arrange
+            var result = new TransferEvidenceNotesViewModel();
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>(Guid.NewGuid(), false));
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>(Guid.NewGuid(), true));
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>(Guid.NewGuid(), true));
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>(Guid.NewGuid(), true));
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>(Guid.NewGuid(), true));
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>(Guid.NewGuid(), true));
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>(Guid.NewGuid(), true));
+            result.SelectedEvidenceNotePairs.Add(new GenericControlPair<Guid, bool>(Guid.NewGuid(), false));
+
+            //act
+            var validationResults = result.Validate(new ValidationContext(result));
+
+            //assert
+            validationResults.Should().Contain(v =>
+                v.ErrorMessage.Equals("You cannot select more than 5 notes")
```

Comment:
I cannot see that message in the CC, only can find "Select up to 5 notes", I guess it's different

Reply:
Have left this as can be empty at the moment as in theory the user could filter out all the results when we get to that bit.

Reply:
okay